gui/screens/settings.py

```diff
@@ -110,7 +111,8 @@ def bio(self, value: str):
     def social_links(self) -> dict:
         self._scroll_view.vertical_scroll_to(self._add_more_links_label)
         links = {}
-        for link_name in walk_children(driver.waitForObjectExists(self._links_list.real_name, configs.timeouts.UI_LOAD_TIMEOUT_MSEC)):
+        for link_name in walk_children(
+                driver.waitForObjectExists(self._links_list.real_name, configs.timeouts.UI_LOAD_TIMEOUT_MSEC)):
```

Comment:
self.links_list.object doesn't work here?

Reply:
i have not changed anything there, i just fixed the formatting the way PyCharm suggested , that's all @StateOf-Vlado